src/main/webapp/index.html

```diff
@@ -5,6 +5,11 @@
     <title>Roomies Home</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <form action="/comments" method="POST">
+      <input name="listingid" placeholder="listing id" required/>
+      <input name="comment" placeholder="comment" required/>
+      <input type="submit" value="Post comment"/>
+    </form>
```

Comment:
Should this be committed? This looks like it is there just to test locally

Reply:
Thanks for catching that, yeah it was just for local tests